docs/adr/0048-notification-setting.md

```diff
@@ -0,0 +1,19 @@
+# 48. Notification setting
+
+Date: 2024-06-04
+
+## Status
+
+Accepted
+
+## Context
+
+- User need a way to tell and change notification configuration in-app
+- However, no app is allowed directly change the device's settings by design
+- With Gmail, Google will redirect users to system settings if they desire to change it
+
+## Decision
+Twake Mail will follow the method Google made for Gmail
```

Comment:
we should not say that we were driven by other company

Reply:
Updated